pkg/labels/labels.go

```diff
@@ -59,8 +76,58 @@ func (ls Labels) String() string {
 		b.WriteString(strconv.Quote(l.Value))
 	}
 	b.WriteByte('}')
+	ret = b.String()
+	b.Reset()
+	bufPool.Put(b)
+	return ret
+}
+
+func (ls Labels) NoRenderString() string {
+	var ret string
+	b := bufPool.Get().(*bytes.Buffer)
+
+	b.WriteByte('{')
+	for i, l := range ls {
+		if i > 0 {
+			b.WriteByte(',')
+			b.WriteByte(' ')
+		}
+		b.WriteString(l.Name)
+		b.WriteByte('=')
+		b.WriteByte('"')
+		b.WriteString(l.Value)
+		b.WriteByte('"')
+	}
+	b.WriteByte('}')
+	ret = b.String()
+	b.Reset()
+	bufPool.Put(b)
+	return ret
+}
 
-	return b.String()
+func yoloString(b []byte) string {
+	return *((*string)(unsafe.Pointer(&b)))
+}
+
+func (ls Labels) YoloString() string {
+	b := yoloPool.Get().(*bytes.Buffer)
+	b.Reset()
+
+	b.WriteByte('{')
+	for i, l := range ls {
+		if i > 0 {
+			b.WriteByte(',')
+			b.WriteByte(' ')
+		}
+		b.WriteString(l.Name)
+		b.WriteByte('=')
+		b.WriteByte('"')
+		b.WriteString(l.Value)
+		b.WriteByte('"')
+	}
+	b.WriteByte('}')
+	yoloPool.Put(b)
+	return yoloString(b.Bytes())
```

Comment:
This isn't a Yolostring, there's nothing backing it that's hanging around. This is just unsafe as b has been put back in the pool.